images/dotnet/README.md

```diff
@@ -26,4 +26,67 @@ docker pull cgr.dev/chainguard/dotnet:latest
 <!--getting:end-->
 
 <!--body:start-->
+## Variants
+We have two images for both sdk and runtime available: a `dotnet-sdk:latest-dev` and a `dotnet-runtime:latest-dev` variant that contains `dotnet` and a shell.
+
+These images are available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/dotnet-sdk:latest
+docker pull cgr.dev/chainguard/dotnet-sdk:latest-dev
+
+docker pull cgr.dev/chainguard/dotnet-runtime:latest
+docker pull cgr.dev/chainguard/dotnet-runtime:latest-dev
+```
+
+## Usage
+
+The dotnet-sdk image can be used directly for simple cases, or with a multi-stage build using the dotnet-sdk as the builder and dotnet-runtime as the final target container.
+
+To get started, go to your current dotnet application directory (or where you house your dotnet applications) and execute the following command. This command should be able to detect the dotnet project in your directory and create a base for the docker initialization.
+
+```docker init```
+
+This command should create the following files.
+
+```
+Dockerfile
+compose.yaml
+README.Docker.md
+.dockerignore
+```
+
+After the files have been created, replace the contents within the created Dockerfile with the following
+
+```Dockerfile
+FROM cgr.dev/chainguard/dotnet-sdk:latest AS build
+
+COPY --chown=nonroot:nonroot . /source
+
+# If your project resides in a sub directory, make sure you are pointing to that directory. ex: If your project resided in a directory called 'app', you would set the destination to /source/app
+WORKDIR /source
+
+RUN dotnet publish --use-current-runtime --self-contained false -o Release
+
+# If you are running an ASPNET project, you can instead pull our 
```

Comment:
I think this comment got cut off somehow. Should it be a multi-line comment with a note to substitute in the aspnet-runtime image?

Reply:
Thank you for catching this. Will fix right now